src/main/java/kubatech/api/enums/GTVoltageIndex.java

```diff
@@ -0,0 +1,22 @@
+package kubatech.api.enums;
```

Comment:
Such constants should probably live [here](https://github.com/GTNewHorizons/GT5-Unofficial/blob/master/src/main/java/gregtech/api/enums/TierEU.java).